src/api.ts

```diff
@@ -97,6 +97,21 @@ export async function updateUser(user: User, options: AuthenticatedHttpOptions):
   return response.data;
 }
 
+export type Role = 'TRUSTED_PATIENT' | 'CLINICIAN';
+
+export interface CreateUserCommand {
+  email: string;
+  roles: Role[];
+}
+
+export async function createUsers(
+  command: CreateUserCommand[],
+  options: AuthenticatedHttpOptions
+): Promise<User[]> {
+  const response = await authenticated(options.token).post('/api/v1/users', command);
```

Comment:
Do we need the cancel token functionality here? I see it being supported for some queries, but not being used.

Reply:
I'd support it. If you use the common hook later it'll be used. Also, you still probably want cancellation in your hook above.

Reply:
Will add it when we'll generalise the common hook to support creation in addition to retrieval as well (at the moment, it fetches on initialisation, so needs a few changes).